tests/unit_tests/restruct_poc/test_yaml_parser.cpp

```diff
@@ -0,0 +1,61 @@
+#include <filesystem>
+
+#include <gtest/gtest.h>
+#include <yaml-cpp/yaml.h>
+
+#include "test_utilities.hpp"
+
+#include "src/utilities/scripts/windio_mapped_structs.hpp"
+
+namespace openturbine::restruct_poc::tests {
```

Comment:
If you want to avoid the need for filesystem logic, you can store the contents of the yaml file in a string (make a test_yaml_parser.hpp with a variable `std::string IEA15File = "..."`).  Then `YAML_::Load(IEA15File)` should parse it.

Reply:
I gave it a shot -- looks like the string literal is too long for the liking of the compiler. I'll defer to you on the best course of action here.

<img width="1347" alt="image" src="https://github.com/user-attachments/assets/bb02fba3-0e25-4e27-988c-1a1030687158">

Reply:
No problem.  I might play with it if I feel froggy, but it's good to go for now.